crates/cxx-qt-gen/src/generator/cpp/externcxxqt.rs

```diff
@@ -12,12 +12,14 @@ use crate::{
 use std::collections::BTreeSet;
```

Comment:
i think there was a book page that said casting only works on RustQt, but now it works on both :-D